.github/workflows/test-semgrep-pro.yaml

```diff
@@ -0,0 +1,72 @@
+name: test-semgrep-pro
+
+on:
+  workflow_dispatch:
+    inputs:
+      artifact-name:
+        required: true
+        type: string
+        description: Name (key) to use when uploading the docker image tarball as a artifact
+      repository-name:
+        required: true
+        type: string
+        description: The repository/name of the docker image to push, e.g., returntocorp/semgrep
+  workflow_call:
+    inputs:
+      artifact-name:
+        required: true
+        type: string
+        description: Name (key) to use when uploading the docker image tarball as a artifact
+      repository-name:
+        required: true
+        type: string
+        description: The repository/name of the docker image to push, e.g., returntocorp/semgrep
+
+jobs:
+  test-semgrep-pro-with-pr:
+    name: Test Semgrep Pro Engine with PR version of Semgrep
+    if: ${{ github.event.pull_request.number != null }}
```

Comment:
There's probably better ways to do this, I'll include a short example

Reply:
https://github.com/returntocorp/semgrep/blob/46bb38e789dce4ac7f4f45cfadf9acbd261b80e1/.github/workflows/release.yml#L35

Try to do something like this, but have the output be the desired tag. Either something of the form `pr-123` or `develop` if you're not on a PR.

Reply:
You can set an output like `docker-tag` and use it as suggested below.